tests/local_server_setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+# Create the local's server directory if necessary
+if [[ ! -d local_server ]]; then
+    mkdir local_server
+fi
+
+cd local_server
+
+# Clone repository if necessary
+if [[ ! -d linux ]]; then
+    # Clone a big repository
+    git clone https://github.com/torvalds/linux
+
+    cd linux
+
+    # Checkout the 5.9 release
+    git checkout v5.9
```

Comment:
Why do we need to do a checkout?

Reply:
So that we keep a stable and deterministic workload. If the linux kernel grows or changes (and it will), then the test will not download the same files. By doing a checkout we're sure to keep downloading the same files over and over even when re-running tests on a fresh clone much later

Reply:
Ok, smart